apps/web/components/DisputeButton.tsx

```diff
@@ -468,7 +464,7 @@ const DisputeMessage = ({
         )}
       </div>
       <div className="chat-bubble shadow-lg bg-neutral-200">
-        {dispute.metadata?.reason ?? disputeMetadata?.reason}
```

Comment:
Metadata can be undefined until the subgraph index the ipfs data source so we need to fallback to the disputeMetadata reason in this case

Reply:
need me to do something here Goss ??